lib/src/Uncertainty/Bayesian/GaussianLinearCalibration.cxx

```diff
@@ -191,10 +191,10 @@ Matrix GaussianLinearCalibration::getGradientObservations() const
   return gradientObservations_;
 }
 
-/* Candidate accessor */
-Point GaussianLinearCalibration::getCandidate() const
+/* ParameterMean accessor */
+Point GaussianLinearCalibration::getParameterMean() const
 {
```

Comment:
so its not deprecated but removed, contrary to what is advertised in the changelog
(deprecated means still there for 1.21, then removed in a later version)

Reply:
Fixed at 4e05a27d6